src/api.test.ts

```diff
@@ -209,6 +209,12 @@ describe('enhancedFetch', () => {
       },
     )
   })
+
+  it('should return some result from other Response methods', async () => {
+    const response = await subject.enhancedFetch('data:text/plain;,foo')
+    const blob = await response.blob()
+    expect(await blob.text()).toEqual('foo')
+  })
```

Comment:
Even though I wrote this test, it was successful, with or without the fix. 🤔

Reply:
Maybe there are cases when `blob()` is dependent of the `Response` state, and sometimes not 🤔